loaders/src/glTF/2.0/Extensions/KHR_quantized_geometry.ts

```diff
@@ -0,0 +1,34 @@
+import { IGLTFLoaderExtension } from "../glTFLoaderExtension";
+import { GLTFLoader } from "../glTFLoader";
+
+const NAME = "KHR_quantized_geometry";
+
+/**
+ * [Specification](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_quantized_geometry)
+ */
+export class KHR_quantized_geometry implements IGLTFLoaderExtension {
+    /**
+     * The name of this extension.
+     */
+    public readonly name = NAME;
+
+    /**
+     * Defines whether this extension is enabled.
+     */
+    public enabled: boolean;
+
+    private _loader: GLTFLoader;
+
+    /** @hidden */
+    constructor(loader: GLTFLoader) {
+        this._loader = loader;
+        this.enabled = this._loader.isExtensionUsed(NAME);
+    }
+
+    /** @hidden */
+    public dispose() {
+        delete this._loader;
+    }

```

Comment:
Can be removed once the private _loader field is removed.

Reply:
I've tried to remove this actually and I can't - the TypeScript compiler (iirc) requires that there's an implementation of dispose. It can be empty though, I'll fix this.

Reply:
Ahh, yes, the interface function is required.